src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.navigate {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-right: 50px;
+}
+
+.logo-img {
+  padding: 10px;
```

Comment:
for this kind of indentation you better use margins, not paddings

Reply:
ok, done